### PR TITLE
Added accessibility to my portfolio

### PR DESCRIPTION
**Pull request by oluyaratosin123**
In this project we did the following:
Made sure there were no linter errors.
Evaluated accessibility and made the necessary changes.

Page titles:no issue found

Image text alternatives
Upon inspection, the alternative text was not descriptive enough the have been adjusted on the following lines.
Social Media icons on Lines - 90, 95, 100, 105, 110
Social Media icons on Lines- , 288, 293, 298, 303,308

Text headings
There were more than one h1 headings. As such we changed the headings to h2 and h4 and adjusted the CSS to match. The lines are reflected below.
HTML Lines - 321,352,371

Color contrast
There were some texts in the input(contact section) that can't be read by an elderly person so we added an hover to the input with background color of a darker blue and white text
CSS Lines: 901, 902, 917, 918, 940, 944

Resize: No issues found.

Interaction: No issues found.

Moving content: No issues found.

Multimedia: No issues found.

The basic structure of the page: No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -431,6 +431,7 @@ <h2 class="form-title">Contact me</h2>
               <textarea
```
Comment (D4nielJ) [inline index.html:431]: [OPTIONAL] You can consider adding descriptions to the labels of your form. You can hide the text in the labels with styles in your CSS stylesheet. This way they will be only accessible to screen readers only. [Read more about it here.](https://a11y-guidelines.orange.com/en/web/components-examples/accessible-hiding/)

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @oluyaratosin123, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (oluyaratosin123) [reply index.html:431]: Hello so I've done the label and its hidden and only available for screen readers only using `sr-only`
Is there any other problem with my project?

File: index.html [hunk 2]
```diff
@@ -87,27 +87,27 @@ <h1 class="title1">
             <ul class="social-media-icons">
               <li>
```
[the remark this answers is not in the document]
Comment (GzimAsani) [reply index.html:88]: - [ ] Please address this requirement asked by the previous reviewer

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi **Team**                         
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the gitflow :+1: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oluyaratosin123** [comment]
How do I do this please

**Comment by GzimAsani** [comment]
> How do I do this please

Take a step back and read this in the requirements --> [Read](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md)

**Comment by oluyaratosin123** [comment]
I have but still don't understand please can you explain

**Comment by oluyaratosin123** [comment]
Is it just that i'm addressing?

**Comment by GzimAsani** [comment]
> I have but still don't understand please can you explain

For this project, you should write a report(In your PR description) about the accessibility of your portfolio, based on the Easy Checks(**you have it in your project description**) – A First Review of Web Accessibility article, listing the accessibility points you checked and writing a conclusion summarizing the results of the evaluation.

**Comment by oluyaratosin123** [comment]
Please you may need to help me on this

**Comment by oluyaratosin123** [comment]
This is my last review

**Review by mikethreels — APPROVED** [review]
Hi Team, 

***Your project is complete!***
There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**